Makefile

```diff
@@ -264,10 +269,10 @@ build-cli:
 	cd ./apps/electionguard-cli && dotnet restore
 	dotnet build -c $(TARGET) $(ELECTIONGUARD_APP_CLI_DIR)/ElectionGuard.CLI.sln /p:Platform=$(PROCESSOR)
 
-build-ui: build-maccatalyst build-netstandard
+build-ui:
 	@echo 🖥️ BUILD UI $(OPERATING_SYSTEM) $(PROCESSOR) $(TARGET)
-	cd ./src/electionguard-ui && dotnet restore
-	dotnet build -c $(TARGET) ./src/electionguard-ui/ElectionGuard.UI.sln /p:Platform=$(PROCESSOR)
+	cd $(ELECTIONGUARD_APP_ADMIN_DIR) && dotnet restore
+	dotnet build -c $(TARGET) $(ELECTIONGUARD_APP_ADMIN_DIR)/ElectionGuard.UI.sln /p:Platform=$(PROCESSOR) /p:APPCENTER_SECRET_UWP=$(APPCENTER_SECRET_UWP) /p:APPCENTER_SECRET_MACOS=$(APPCENTER_SECRET_MACOS)
```

Comment:
adding these secrets for builds. if they are null or not set then it just doesnt get loaded